charts/bindplane/templates/bindplane-jobs.yaml

```diff
@@ -235,6 +235,12 @@ spec:
             - name: BINDPLANE_LDAP_TLS_CA
               value: /ldap-ca.crt
             {{- end }}
+            {{- if .Values.auth.ldap.tls.clientKeyPair.secret }}
+            - name: BINDPLANE_LDAP_TLS_CERT
+              value: /ldap-client.crt
+            - name: BINDPLANE_LDAP_TLS_KEY
+              value: /ldap-client.key
+            {{- end }}
```

Comment:
These file paths can be whatever we want, as long as they match the `mountPath` used in the `volumeMounts` below.